.config/nextest.toml

```diff
@@ -1,11 +1,25 @@
+[test-groups]
+against-tilt = { max-threads = 1 }
+
 [profile.default]
 default-filter = "not test(/test_k8s_integration_/)"
 
 [profile.k8s_integration]
 default-filter = "test(/test_k8s_integration_/)"
 
+[[profile.k8s_integration.overrides]]
+filter = "all()"
+test-group = "against-tilt"
+
 [profile.ci.junit]
 path = "junit.xml"
 
+[profile.ci_k8s_integration]
+default-filter = "test(/test_k8s_integration_/)"
+
+[[profile.ci_k8s_integration.overrides]]
```

Comment:
Maybe we ought to create a new group here? As when we expand the test cases to make rust tests drive tilt this will make it slow. Fine with bookmarking that and solving later too

Reply:
I agree. Created a new issue to think about integration testing in Rust more holistically: https://github.com/chroma-core/hosted-chroma/issues/2178